App/Redux/IpfsNodeRedux.js

```diff
@@ -31,12 +31,18 @@ export const INITIAL_STATE = Immutable({
   threads: {
     default: {
       querying: false,
-      hashes: [],
+      hashes: {
```

Comment:
I guess this would be cleaner if `hashes` was an array of objects, those objects looking like:
```
{ 
  hash: string,
  uri: string
```
Also seems like the 'path' should just be the uri to the ipfs directory, including the protocol (http, etc)